Assets/LeapMotion/Scripts/Utils/ValueMappingExtensions.cs

```diff
@@ -0,0 +1,125 @@
+﻿using UnityEngine;
+
+namespace Leap.Unity {
+
+  public static class ValueMappingExtensions {
```

Comment:
Well it's an extension, so being verbose is fine :D you never actually see the name.